src/less/button/button.less

```diff
@@ -263,7 +263,6 @@ a.fake-btn--secondary[aria-disabled="true"] {
 
 button.btn--tertiary,
 a.fake-btn--tertiary {
-    border-width: 2px;
```

Comment:
I think tertiary buttons have a default of 2px
The split button tertiary should only have 1px

Reply:
Yea, was running through the Percy build and saw that. Working on it.